admin/plugins.php

```diff
@@ -0,0 +1,393 @@
+		$plugin = (array) $plugin;
+	}
+
+	return $plugin;
+}
+
+/**
+ * Returns an array of WPP standalone plugins.
+ *
+ * @since n.e.x.t
+ *
+ * @return array of wpp standalone plugins as slugs.
+ */
+function perflab_get_standalone_plugins() {
+	return array(
+		'webp-uploads',
+		'performant-translations',
+		'dominant-color-images',
+	);
+}
+
+/**
+ * Renders plugin UI for managing standalone plugins within PL Settings screen.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_plugins_ui() {
+	require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+	require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+	$standalone_plugins = array();
+	foreach ( perflab_get_standalone_plugins() as $managed_standalone_plugin_slug ) {
+		$standalone_plugins[ $managed_standalone_plugin_slug ] = array(
+			'plugin_data' => perflab_query_plugin_info( $managed_standalone_plugin_slug ),
+		);
+	}
+
+	if ( empty( $standalone_plugins ) ) {
+		return;
+	}
+	?>
+	<div class="wrap">
+		<h1><?php esc_html_e( 'Performance Plugins', 'performance-lab' ); ?></h1>
+		<p><?php esc_html_e( 'The following standalone performance plugins are available for installation.', 'performance-lab' ); ?></p>
+		<div class="wrap">
+			<form id="plugin-filter" method="post">
+				<div class="wp-list-table widefat plugin-install wpp-standalone-plugins">
+					<h2 class="screen-reader-text"><?php esc_html_e( 'Plugins list', 'default' ); ?></h2>
+					<div id="the-list">
+						<?php
+						foreach ( $standalone_plugins as $standalone_plugin ) {
+							perflab_render_plugin_card( $standalone_plugin['plugin_data'] );
+						}
+						?>
+					</div>
+				</div>
+			</form>
+		</div>
+		<div class="clear"></div>
+	</div>
+	<?php
+}
+
+/**
+ * Renders individual plugin cards.
+ *
+ * This is adapted from `WP_Plugin_Install_List_Table::display_rows()` in core.
+ *
+ * @since n.e.x.t
+ *
+ * @see WP_Plugin_Install_List_Table::display_rows()
+ * @link https://github.com/WordPress/wordpress-develop/blob/0b8ca16ea3bd9722bd1a38f8ab68901506b1a0e7/src/wp-admin/includes/class-wp-plugin-install-list-table.php#L467-L830
+ *
+ * @param array $plugin_data Plugin data from the WordPress.org API.
+ */
+function perflab_render_plugin_card( array $plugin_data ) {
+	// If no plugin data is returned, return.
+	if ( empty( $plugin_data ) ) {
+		return;
+	}
+
+	// Remove any HTML from the description.
+	$description = wp_strip_all_tags( $plugin_data['short_description'] );
+	$title       = $plugin_data['name'];
+
+	/** This filter is documented in wp-admin/includes/class-wp-plugin-install-list-table.php */
+	$description = apply_filters( 'plugin_install_description', $description, $plugin_data );
+	$version     = $plugin_data['version'];
+	$name        = wp_strip_all_tags( $title . ' ' . $version );
+	$author      = $plugin_data['author'];
+	if ( ! empty( $author ) ) {
+		/* translators: %s: Plugin author. */
+		$author = ' <cite>' . esc_html( sprintf( __( 'By %s', 'default' ), $author ) ) . '</cite>';
```

Comment:
This currently causes HTML tags to be escaped when they shouldn't. Also note that either way there should not be any escaping here as escaping should _only_ be done right when outputting. Here we're just putting the string in a variable, i.e. it should not be escaped at this point.

The `$author` string is eventually run through `wp_kses_post()` further down, so we can simply remove this and call it a day.
```suggestion
		$author = ' <cite>' . sprintf( __( 'By %s', 'default' ), $author ) . '</cite>';
```

Reply:
@felixarntz I believe it was just this one instance, now corrected. Apologies for not committing your suggestion, I'd already committed that before seeing the above.

Reply:
All good!